arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -120,15 +116,18 @@ public Bytes getCodeByMetadata(final ContractMetadata metadata) {
     return getChunkByMetadata(metadata).map(RomChunk::byteCode).orElse(Bytes.EMPTY);
   }
 
+  // TODO: it would maybe make more sense to only implement traceContextEnter
```

Comment:
hum ... not sure. Because when we enter a context, ROMLEX is triggered by the call / create section. Here ROMLEX is triggered by the Besu hook, so it's quite different.